browser/edge/find.go

```diff
@@ -1,22 +1,13 @@
-//go:build windows
-
 package edge
 
 import (
-	"errors"
-	"os"
-	"path/filepath"
-
 	"github.com/browserutils/kooky"
 	"github.com/browserutils/kooky/internal/chrome"
 	"github.com/browserutils/kooky/internal/chrome/find"
 	"github.com/browserutils/kooky/internal/cookies"
-	"github.com/browserutils/kooky/internal/ie"
-	_ "github.com/browserutils/kooky/internal/ie/find"
```

Comment:
leave the support for old edge  in. I would like to support analyzing old directory structures in the future.